IFC4x3/PropertyConstants/n/NEWBUILD_3_S218crb2exItJY2XPrej.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NEWBUILD_3_S218crb2exItJY2XPrej" Name="NEWBUILD" UniqueId="fe702048-9b59-42a3-b4b7-4e20a1675a2d" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NEWBUILD_3_S218crb2exItJY2XPrej" Name="NEWBUILD" UniqueId="fe702048-9b59-42a3-b4b7-4e20a1675a2d">
+	<Documentation>Project responsible for the development of a new build facility</Documentation>
```

Comment:
'build facility' -> 'built facility'?

Reply:
'new build' is a phrase as te property name suggests.